pkg/base/testdata/raft_config

```diff
@@ -2,19 +2,19 @@ echo
 ----
 (base.RaftConfig) {
  RaftTickInterval: (time.Duration) 200ms,
- RaftElectionTimeoutTicks: (int) 15,
+ RaftElectionTimeoutTicks: (int) 10,
  RaftHeartbeatIntervalTicks: (int) 5,
- RangeLeaseRaftElectionTimeoutMultiplier: (float64) 3,
+ RangeLeaseRaftElectionTimeoutMultiplier: (float64) 2.5,
  RangeLeaseRenewalFraction: (float64) 0.5,
  RaftLogTruncationThreshold: (int64) 16777216,
  RaftProposalQuota: (int64) 8388608,
  RaftMaxUncommittedEntriesSize: (uint64) 16777216,
  RaftMaxSizePerMsg: (uint64) 32768,
  RaftMaxCommittedSizePerReady: (uint64) 67108864,
  RaftMaxInflightMsgs: (int) 128,
- RaftDelaySplitToSuppressSnapshotTicks: (int) 245
+ RaftDelaySplitToSuppressSnapshotTicks: (int) 230
 }
-RaftElectionTimeout: 3s
```

Comment:
Would be good if this also printed RaftHeartbeatIntervalTicks*RaftTickInterval (i.e. the raft "heartbeat interval").

Reply:
Done.